src/components/AddressBookSelector.tsx

```diff
@@ -89,29 +90,31 @@ const AddressBookSelector = forwardRef(
     }, [homeNav])
 
     return (
-      <BottomSheetModalProvider>
-        <Box flex={1} {...boxProps}>
-          <BottomSheetModal
-            ref={bottomSheetModalRef}
-            index={0}
-            backgroundStyle={backgroundStyle}
-            backdropComponent={renderBackdrop}
-            snapPoints={snapPoints}
-            handleStyle={sheetHandleStyle}
-            onDismiss={handleDismiss}
-          >
-            <ContactsList
-              showMyAccounts
-              hideCurrentAccount={hideCurrentAccount}
-              onAddNew={handleAddNewContact}
-              handleContactSelected={handleContactSelected}
-              address={address}
-              insideBottomSheet
-            />
-          </BottomSheetModal>
-          {children}
-        </Box>
-      </BottomSheetModalProvider>
+      <Box flex={1} {...boxProps}>
+        <Portal>
```

Comment:
Why is the Portal now needed?

Reply:
Same reason as above. Since we added the `TabBarNavigator` we need it to appear above the tabs

Reply:
Same reason as above. Since we added the `TabBarNavigator` we need it to appear above the tabs